eng/versioning/version_client.txt

```diff
@@ -62,9 +62,9 @@ com.azure:azure-communication-common;1.1.2;1.2.0-beta.1
 com.azure:azure-communication-common-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-communication-sms;1.1.1;1.2.0-beta.1
 com.azure:azure-communication-identity;1.1.8;1.2.0-beta.2
-com.azure:azure-communication-phonenumbers;1.0.8;1.1.0-beta.5
-com.azure:azure-communication-networktraversal;1.1.0-beta.2;1.1.0-beta.3
-com.azure:azure-containers-containerregistry;1.0.2;1.1.0-beta.2
+com.azure:azure-communication-phonenumbers;1.0.10;1.1.0-beta.5
+com.azure:azure-communication-networktraversal;1.0.0;1.1.0-beta.3
```

Comment:
Was anything using the beta dependency?

Reply:
Nobody is depending on it. However, I will still undo this since it is not needed for my change and it may be a very conscious call from the service.